yabt/buildcontext.py

```diff
@@ -279,8 +279,9 @@ def make_fail_callback(target: Target):
             def fail_notifier(ex):
                 """Mark target as failed, taking it and ancestors
                    out of the queue"""
-                sys.stdout.write(ex.stdout.decode('utf-8'))
-                sys.stderr.write(ex.stderr.decode('utf-8'))
+                if isinstance(ex, CalledProcessError):
```

Comment:
Please add here a comment pointing to the (still non-extsing?) issue discussion the need to distinguish "failed to build target" and "failed to run"